contracts/account-history-contract/src/entry_point/query.rs

```diff
@@ -14,5 +15,19 @@ pub fn query(deps: Deps<ElysQuery>, env: Env, msg: QueryMsg) -> StdResult<Binary
             let resp = querrier.accounts(pagination)?;
             resp
         }),
+        All {} => to_json_binary(&{
+            let list = HISTORY
+                .prefix_range(deps.storage, None, None, Order::Ascending)
+                .filter_map(|res| res.ok())
+                .collect::<Vec<(String, Vec<AccountSnapshot>)>>();
+            list
+        }),
+        LastSnapshot { user_address } => to_json_binary(&{
+            let snapshots = HISTORY.load(deps.storage, &user_address)?;
+            match snapshots.last().cloned() {
+                Some(expr) => expr,
+                None => return Err(StdError::not_found("account snapshot")),
+            }
+        }),
```

Comment:
@politeWall we actually need to have another query that returns the list of liquid assets and total balance of liquid assets alone based on the latest snapshot